packages/xo-server-sdn-controller/src/index.js

```diff
@@ -760,7 +723,8 @@ class SDNController extends EventEmitter {
       const hosts = filter(pool.$xapi.objects.all, { $type: 'host' })
       await Promise.all(
         map(hosts, async host => {
-          await createTunnel(host, createdNetwork)
+          const hostPif = host.$PIFs.find(_pif => _pif.network === pif.network)
```

Comment:
is it possible that the host don't have a pif on this network ?

Reply:
Maybe, but `createTunnel` already handle if `hostPif` is `undefined`